tests/components/reactionsChart.logic.test.ts

```diff
@@ -0,0 +1,125 @@
+import { endOfDay, parseISO, startOfDay } from "date-fns";
+import { describe, expect, it, vi } from "vitest";
+
+// This tests the core logic that was fixed in the bug
+describe("ReactionsChart handleBarClick logic", () => {
+  // Mock the date functions
+  vi.mock("date-fns", () => ({
+    startOfDay: vi.fn((date: Date) => new Date(date.getFullYear(), date.getMonth(), date.getDate())),
+    endOfDay: vi.fn((date: Date) => new Date(date.getFullYear(), date.getMonth(), date.getDate(), 23, 59, 59, 999)),
+    parseISO: vi.fn((dateString: string) => new Date(dateString)),
+  }));
+
+  // Simulate the handleBarClick function logic that was fixed
+  function createSearchParams(data: any, reactionType: "thumbs-up" | "thumbs-down") {
+    const clickedDate = parseISO(data.date || data.payload?.date);
+
+    // This is the bug fix: use reactionAfter/reactionBefore instead of createdAfter/createdBefore
+    return {
+      reactionAfter: startOfDay(clickedDate),
+      reactionBefore: endOfDay(clickedDate),
+      reactionType,
+      // The bug was using these parameters instead:
+      // createdAfter: startOfDay(clickedDate),
+      // createdBefore: endOfDay(clickedDate),
+    };
```

Comment:
```suggestion
    return {
      reactionAfter: startOfDay(clickedDate),
      reactionBefore: endOfDay(clickedDate),
      reactionType,
    };
```

These don't need to be specified